component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -828,6 +819,27 @@ public Response updateUserById(@Context UriInfo uriInfo,
     return EntityBuilder.getResponse(EntityBuilder.buildEntityProfile(id, uriInfo.getPath(), expand), uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK);
   }
 
+  @GET
+  @Path("onBoarding/{id}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("administrators")
+  @ApiOperation(value = "Send onBoarding email to a specific user", 
+                httpMethod = "GET", 
+                response = Response.class, 
+                notes = "This send onBoarding email to a specific user.")
+  public Response sendOnBoardingEmail(@Context UriInfo uriInfo,
+                                      @ApiParam(value = "User name", required = true) @PathParam("id") String id) throws Exception {
+    UserHandler userHandler = organizationService.getUserHandler();
+    User user = userHandler.findUserByName(id);
+    if (user == null) {
+      throw new WebApplicationException(Response.Status.BAD_REQUEST);
+    }
+    String uri = uriInfo.getBaseUri().toString().substring(0, uriInfo.getBaseUri().toString().lastIndexOf("/"));
+    StringBuilder url = new StringBuilder(uri);
+    sendOnBoardingEmail((UserImpl) user, url);
+    return Response.noContent().build();
```

Comment:
Can you confirm that Response.noContent() return a 200
Is it better than Response.ok().build() ?

Reply:
no , it return 204.
Updated to Response.ok().build() !